### PR TITLE
DOC: improve faq

### PR DESCRIPTION
**Pull request by sappelhoff**
see: https://github.com/autoreject/autoreject/issues/111#issuecomment-527582092

cc @jasmainak

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -10,33 +10,47 @@ Should I apply ICA first or autoreject first?
...
+	>>> reject = get_rejection_threshold(epochs)
+	>>> ica.fit(epochs, reject=reject)
 
-or use autoreject (global)::
+In case you want to fit your ICA on the raw data, you will need an intermediate
+step, because autoreject only works on epoched data. ICA is ignoring the time
+domain of the data, so we can simply turn the raw data into equally spaced
+"fixed length" epochs using ::func::`mne.make_fixed_length_events`::
 
+	>>> epo_duration = 1.0
```
Comment (jasmainak) [inline doc/faq.rst]: let's call this `duration`

Comment (jasmainak) [reply doc/faq.rst]: or even better `tstep` to match the parameter in `ica.fit()`

File: doc/faq.rst [hunk 2]
```diff
@@ -10,33 +10,47 @@ Should I apply ICA first or autoreject first?
...
+	>>> events = mne.make_fixed_length_events(raw, duration=epo_duration)
+	>>> epochs = mne.Epochs(raw, events, tmin=0.0, tmax=epo_duration)
 	>>> reject = get_rejection_threshold(epochs)
 	>>> ica.fit(epochs, reject=reject)
```
Comment (jasmainak) [inline doc/faq.rst]: then pass `tstep` also here

File: doc/faq.rst [hunk 3]
```diff
@@ -10,33 +10,47 @@ Should I apply ICA first or autoreject first?
...
+components, the ICA results can be applied to the raw data::
 
 	>>> ica.exclude = [5, 7]  # exclude EOG components
-	>>> ica.transform(epochs)
+	>>> clean_raw = ica.apply(raw)
+
+After obtaining the ICA cleaned raw data, you may consider making your own
+specific epochs, and applying autoreject (local).
+
+As an alternative to using autoreject (global) before the ICA, and autorejct
```
Comment (jasmainak) [inline doc/faq.rst]: autorejct -> autoreject

**Comment by sappelhoff** [comment]
done @jasmainak

**Comment by jasmainak** [comment]
you know that I'll be notified every time you rebase if you tag me in the commit? :)

anyway, looks good. Merging, thanks @sappelhoff !

**Comment by sappelhoff** [comment]
Yes I know - BUT ... I also happen to know that you are muting several communication channels, and I never know which ones - so I am trying to use several ones if it's urgent :)

**Pull request merged**